src/System.CodeDom/tests/CodeCollections/CodeNamespaceImportCollectionTests.cs

```diff
@@ -5,23 +5,37 @@
 using System.Collections;
 using System.Collections.Generic;
 using System.Collections.Tests;
+using System.Runtime.InteropServices;
 using Xunit;
 
 namespace System.CodeDom.Tests
 {
     public class CodeNamespaceImportCollectionTests : IList_NonGeneric_Tests
     {
+        private bool _isFullFramework => RuntimeInformation.FrameworkDescription.StartsWith(".NET Framework", StringComparison.OrdinalIgnoreCase);
```

Comment:
can we move this property under common and try to use it from there? so we don't have to define it in multiple places